src/pudl/transform/eia.py

```diff
@@ -543,24 +544,15 @@ def harvesting(  # noqa: C901
     mcs = consistency["consistent_ratio"].mean()
     logger.info(f"Average consistency of static {entity} values is {mcs:.2%}")
 
-    if entity == "plants":
-        entity_df = _add_additional_epacems_plants(entity_df)
-        entity_df = _add_timezone(entity_df)
-
-    eia_transformed_dfs[f"{entity}_annual_eia"] = annual_df
-    entities_dfs[f"{entity}_entity_eia"] = entity_df
-    if debug:
-        return entities_dfs, eia_transformed_dfs, col_dfs
-    return (entities_dfs, eia_transformed_dfs)
+    return entity_df, annual_df, col_dfs
 
 
-# TODO (bendnorman): Can this and harvesting() be separate assets?
```

Comment:
I created a wrapper function `boiler_generator_assn_eia860` below that is decorated with `@asset`. Would it be better to just rename this function and decorate it here instead? The wrapper doesn't do much other than apply PUDL dtypes.

Reply:
Yeah, it seems much neater. Glad it's so easy to just decorate existing functions!